### DISCUSSION TITLE
Qiskit Release Frequency

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#4] [Releases & Roadmap] [answered]
I'm opening this discussion for general thoughts on the Qiskit release pace.

Do Qiskit releases happen too often? Not often enough? Just about right? What would be an ideal frequency be in your opinion?

We've heard differing opinions on this so would like to open up a public discussion here, let us know your thoughts!

**Comment by omarcostahamido** [comment]
as long as an update doesn't break any _backward compatibility_ it should be as often as possible.

**Comment by enavarro51** [comment]
I know there are probably manpower issues with getting releases out, but I agree with @omarcostahamido that more often would be better. For a developer who does deprecations in the code, the time from writing the deprecation to getting the deprecations cleaned out can be 6 months or more. That leaves a lot of messy code around for an excessive time.

Likewise I think users who find a bug early after another release currently have an excessive wait for the fix (assuming they don't want to deal with the source.)

**Comment by 1ucian0** [comment, marked as answer]
The [`RFC 0022`](https://github.com/Qiskit/RFCs/blob/master/0020-release_cycle.md) is the answer :)
